### ISSUE TITLE
Doesn't seem to autocomplete when command has a ':'

### ISSUE THREAD

**Issue by jblandry** [f/omelette#8]
I'm trying to autocomplete 
```
mybin scope:task1 arg
mybin scope:task2 arg
mybin scope:task3 arg
```

Omelette autocompletes until `mybin scope:` but doesn't return anything after.

Thanks in advance

**Comment by f** [comment]
Because -unfortunately- Bash/Zsh or Fish doesn't work that way. You cannot use ':' as separator, you have to use spaces.

Simply try in Bash:
```
$ complete -W 'y:a y:b y:c x:a x:b x:c' hello
$ hello x:<tab>
y:a y:b y:c x:a x:b x:c
```

It won't work.

**Comment by jblandry** [comment]
Aaaah crap :(
Ok thanks

**Comment by jblandry** [comment]
Mmmm I've just tried with
```
$ complete -W 'y:abc y:abb y:aba' hello
$ hello y<tab>
```
And it autocompletes to `hello y:ab`

I'm on bash 4.4.12(1)-release on macOS

**Comment by jblandry** [comment]
Ok I've tried other combinations and it does not work. Bash seems to treat ':' as special char and not a common char.

```
$ complete -W 'y:abc y:abb y:aba' hello
$ hello y<tab>      -> hello y:ab
$ hello y:<tab>     -> hello y:y:ab
$ hello y:ab<tab>   -> no joy
```

I will find an alternative thanks

**Comment by yuchi** [comment]
Probably related: http://stackoverflow.com/questions/17035469/autocompleting-a-word-that-contains-a-colon

**Comment by yuchi** [comment]
Also: http://stackoverflow.com/questions/2805412/bash-completion-for-maven-escapes-colon

**Comment by jblandry** [comment]
This solution `COMP_WORDBREAKS=${COMP_WORDBREAKS//:}` works like a charm. Is there a possibility of adding it to omelette ? If so I could make a PR :)

**Comment by f** [comment]
I am waiting for the PR :)

3 May 2017 Çar 15:34 tarihinde Jb <notifications@github.com> şunu yazdı:

**Comment by yuchi** [comment]
Pay attention it's invasive: it modifies the value for all auto completes and doing it in a CLI is a little impolite.
Make sure you don't actually change the user environment.

**Comment by jblandry** [comment]
@yuchi Good call. Changed the PR -> https://github.com/f/omelette/pull/9

**Comment by jblandry** [comment]
Merged #9
